packages/sandbox/src/typeAcquisition.ts

```diff
@@ -211,7 +211,7 @@ const getModuleAndRootDefTypePath = async (packageName: string, config: ATAConfi
 
   const responseJSON = await response.json()
   if (!responseJSON) {
-    return errorMsg(`Could the Algolia JSON was un-parsable for the module '${packageName}'`, response, config)
+    return errorMsg(`Could not get the Algolia JSON was un-parsable for the module '${packageName}'`, response, config)
```

Comment:
this sentence still feels weird, what about:

```suggestion
    return errorMsg(`Could not get the Algolia JSON for the module '${packageName}'`, response, config)
```

(the parsable seems equivalent to could not get?)

Reply:
It's fine, this file is going to get deleted when I mark this PR as for production 👍🏻